extensions/resource-deployment/src/interfaces.ts

```diff
@@ -100,23 +100,26 @@ export interface DeploymentProviderBase {
 
 export type DeploymentProvider = DialogDeploymentProvider | BdcWizardDeploymentProvider | NotebookWizardDeploymentProvider | NotebookDeploymentProvider | WebPageDeploymentProvider | DownloadDeploymentProvider | CommandDeploymentProvider;
 
-export interface WizardInfo {
+export interface BdcWizardInfo {
 	notebook: string | NotebookInfo;
 	type: BdcDeploymentType;
 }
 
 export interface NotebookWizardInfo extends WizardInfoBase {
 	notebook: string | NotebookInfo;
+	runNotebook?: boolean;
+	codeCellInsertionPosition?: number;
+	pages: NotebookWizardPageInfo[];
+	summaryPage: NotebookWizardPageInfo;
 }
 
-export interface WizardInfoBase extends SharedFieldAttributes {
+export interface WizardInfoBase extends InheritableAttributes {
 	taskName?: string;
 	type?: DeploymentType;
-	runNotebook?: boolean;
 	actionText?: string;
 	title: string;
-	pages: NotebookWizardPageInfo[];
-	summaryPage: NotebookWizardPageInfo;
+	pages: PageInfoBase[];
+	summaryPage: PageInfoBase;
```

Comment:
why can't this just be part of pages property?

Reply:
it is a wizard level property whether to generate an automatic summaryPage. I will rename it to make it clearer.